src/utils/domains.ts

```diff
@@ -1,8 +1,16 @@
 import { Id } from '@colony/colony-js';
 
 import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
+import { Colony } from '~types';
+
+import { notNull } from './arrays';
 
 export const getDomainId = (ethDomainId?: number) =>
   ethDomainId === COLONY_TOTAL_BALANCE_DOMAIN_ID || ethDomainId === undefined
     ? Id.RootDomain
     : ethDomainId;
+
+export const getDomain = (colony: Colony, domainId: number) =>
+  colony?.domains?.items
+    .filter(notNull)
+    .find(({ nativeId }) => nativeId === domainId);
```

Comment:
Nice one, I've thought about creating one for colony's tokens similar to this one as this logic is used a lot with the domains and tokens on the dialogs